packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -193,6 +207,20 @@ export class BuiltinLambda extends BaseLambda {
     }
     throw new REOther(showNameMatchDefinitions());
   }
+
+  toCalculator(): Calculator {
+    const longestSignature = maxBy(this.signatures(), (s) => s.length) || [];
+    const autorun = longestSignature.length !== 0;
+    return {
+      fn: this,
+      inputs: longestSignature.map((sig, i) => ({
+        name: `Input ${i + 1}`,
+        type: sig.getName() === "Bool" ? "checkbox" : "text",
+      })),
+      sampleCount: 10000,
```

Comment:
I wanted this to be a bit higher than normal, because these are probably fairly small (just one function). 

That said, as I think about it, maybe it could be a very expensive function. Can change.